azure.yaml

```diff
@@ -25,20 +27,20 @@
       componentVersion: 0.18.1
       version: 1.2.0
   authorities:
-  - name: app-operator
-    version: 1.0.0
-  - name: azure-operator
-    version: 2.9.0
-  - name: cert-operator
-    version: 0.1.0
-  - name: chart-operator
-    version: 0.7.0
-  - name: cluster-operator
-    provider: azure
-    version: 0.23.0
-  date: 2020-01-29T12:00:00Z
-  version: 11.2.0
-- active: false
+    - name: app-operator
```

Comment:
I found out these parts need an indentation. Not sure how it was working before.